README.md

```diff
@@ -194,6 +195,9 @@ Developers
 
 See [CONTRIBUTING.md](CONTRIBUTING.md)
 
+If you like to jump in straight away, see our
+[!["good first issue(s)" open](https://img.shields.io/github/issues-raw/ComputationalRadiationPhysics/picongpu/good%20first%20issue.svg?color=56cbef)](https://github.com/ComputationalRadiationPhysics/picongpu/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
```

Comment:
(not important)
`open "good first issue" issues` is a little weird. I assume `open "good first issues"` was not your choice since our label name is not in the plural, and `open "good first issue"s` is probably wrong or at least even more weird.

Reply:
Btw, maybe `open "good first issue" list`?

Reply:
yes, but the tag is not modifiable to that extent